VERSION.txt

```diff
@@ -1198,7 +1198,7 @@ jetty-9.4.4.v20170414 - 14 April 2017
    encoding
  + 1436 NullPointerException when calling changeSessionId
  + 1439 Allow UNC paths to function as Resource bases
- + 1440 Improve lock contention for low resources scheduling strategy
+ + 1440 Improve lock contention for low resources scheduling strategyType
```

Comment:
Revert this, it was a search/replace gone wrong.

Reply:
Ah ooops.   That's an IDE being too smart!